group_vars/cdhweb/vars.yml

```diff
@@ -20,6 +26,12 @@ node_version: "10"
 # django database backend
 db_backend: "postgresql"
 db_host: "{{ postgres_host }}"
-# image libraries needed for mezzanine image handling
+# postgresql database info
+application_dbuser_name: "{{ vault_db_username }}"
+application_dbuser_password: "{{ vault_db_password }}"
+application_db_name: "{{ vault_db_name }}"
+# Database backup location
+db_backup_path: '/home/{{ deploy_user }}/pre-{{ version }}-{{ short_hash }}.sql'
```

Comment:
Nice, better than putting in tmp. Should it go in a subdirectory, maybe?

Reply:
Probably a good idea, just need to add to the role to try to create the directory it if it doesn't exist

Reply:
Added some logic to do this to the backup task, will test on next deploy